pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -474,6 +478,11 @@ func (c *MaxPDVolumeCountChecker) predicate(pod *v1.Pod, meta PredicateMetadata,
 		return true, nil, nil
 	}
 
+	// If a plugin has been migrated to a CSI driver, defer to the CSI predicate.
```

Comment:
Does the CSI predicate already implement per node volume limits? I don't see that update in this PR.